src/Dapr.Client/DaprClient.cs

```diff
@@ -324,5 +324,17 @@ public async ValueTask<StateEntry<TValue>> GetStateEntryAsync<TValue>(string sto
             string key,
             Dictionary<string, string> metadata = default,
             CancellationToken cancellationToken = default);
+
+        /// <summary>
+        /// Gets all secret values from the secret store.
+        /// </summary>
+        /// <param name="storeName">Secret store name.</param>
+        /// <param name="metadata">An key/value pair that may be consumed by the secret store.  This depends on the secret store used.</param>
```

Comment:
Can you update this to be the same as the other `metadata` parameters on this class? We did a review pass here after you created this PR.

Reply:
done